app/containers/ReactionsList/ReactionsTabBar.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { Text, Pressable, View, ScrollView } from 'react-native';
+
+import Emoji from '../message/Emoji';
+import { useTheme } from '../../theme';
+import { IReaction } from '../../definitions';
+import { TGetCustomEmoji } from '../../definitions/IEmoji';
+import I18n from '../../i18n';
+import styles, { MIN_TAB_WIDTH } from './styles';
+
+interface ITabBarItem {
+	baseUrl: string;
+	getCustomEmoji: TGetCustomEmoji;
+	tab: IReaction;
+	index: number;
+	goToPage?: (index: number) => void;
+}
+interface IReactionsTabBar {
+	baseUrl: string;
+	getCustomEmoji: TGetCustomEmoji;
+	activeTab?: number;
+	tabs?: IReaction[];
+	goToPage?: (index: number) => void;
+	width: number;
+}
+
+const TabBarItem = ({ tab, index, goToPage, baseUrl, getCustomEmoji }: ITabBarItem) => {
+	const { colors } = useTheme();
+	return (
+		<Pressable
+			key={tab.emoji}
+			onPress={() => {
+				goToPage?.(index);
+			}}
+			style={({ pressed }: { pressed: boolean }) => ({
+				opacity: pressed ? 0.7 : 1
+			})}
+			testID={`tabBarItem-${tab.emoji}`}
+		>
+			<View style={styles.tabBarItem}>
+				{tab._id === 'All' ? (
+					<Text style={[styles.allTabItem, { color: colors.auxiliaryTintColor }]}>{I18n.t('All')}</Text>
+				) : (
+					<>
+						<Emoji
+							content={tab.emoji}
+							standardEmojiStyle={styles.standardEmojiStyle}
+							customEmojiStyle={styles.customEmojiStyle}
+							baseUrl={baseUrl}
+							getCustomEmoji={getCustomEmoji}
+						/>
+						<Text style={[styles.reactionCount, { color: colors.auxiliaryTintColor }]}>{tab.usernames.length}</Text>
+					</>
+				)}
+			</View>
+		</Pressable>
+	);
+};
+
+const ReactionsTabBar = ({ tabs, activeTab, goToPage, baseUrl, getCustomEmoji, width }: IReactionsTabBar): React.ReactElement => {
+	const tabWidth = tabs && Math.max(width / tabs.length, MIN_TAB_WIDTH);
```

Comment:
It's 18% on Figma instead of calculated. Can you try that instead?

Reply:
But it'll look like this: 

* <img width="354" alt="Screenshot 2022-08-26 at 12 43 49 AM" src="https://user-images.githubusercontent.com/77742477/186749528-563bdc0e-7924-4846-8971-fdfea468e239.png">

instead of 
* <img width="359" alt="Screenshot 2022-08-26 at 12 41 12 AM" src="https://user-images.githubusercontent.com/77742477/186749551-5a303cea-15d6-4655-81ae-71f13be18c70.png">

IMO, the second design looks better. I can change it to the other one if you want. Anyways I've fixed the tab width issue on landscape.